truss/cli/cli.py

```diff
@@ -471,17 +444,33 @@ def chains_init(
     user_friendly_path = os.path.join(target_directory, FILENAME)
     rich.print(f"Creating {user_friendly_path}...\n")
 
-    os.mkdir(abs_path)
+    source_code = _load_example_chainlet_code()
+
+    if not os.path.exists(abs_path):
+        os.mkdir(abs_path)
     with open(filename, "w") as f:
-        f.write(HELLO_PY)
+        f.write(source_code)
 
     rich.print(
         "Next steps:\n",
-        f"Run `python {user_friendly_path}` locally by adding `chains.run_local()` to your main.py file\n",
+        f"Run `python {user_friendly_path}` to execute the code locally\n",
```

Comment:
Optional: you could add some fun emojis here ;)

Reply:
added!